expertsystem/ui/_system_control.py

```diff
@@ -219,7 +265,11 @@ def _remove_qns_from_graph(
     return graph_copy
 
 
-def _check_equal_ignoring_qns(ref_graph, solutions, ignored_qn_list):
+def _check_equal_ignoring_qns(
+    ref_graph: StateTransitionGraph,
+    solutions: List[StateTransitionGraph],
+    ignored_qn_list: List[str],
```

Comment:
Those should be `List[Union[StateQuantumNumberNames, InteractionQuantumNumberNames]]`

Reply:
36b6f19d88e4fad31caf433ea135f3aa4465501c